src/components/ProfileInformation/ProfileInformation.module.css

```diff
@@ -174,3 +174,7 @@
 :global(.ui.dropdown) :global(.menu>.item).copyItem > svg {
   margin-right: 11px;
 }
+
+.userNameLastPart {
+  color: #A09BA8
```

Comment:
shouldn't we use library colors?

Reply:
We don't have that color set as a variable anywhere else, I've set it in the `index.css` file of the site so we can use it whenever it comes up.